plugins/hydra_ax_sweeper/tests/test_ax_sweeper_plugin.py

```diff
@@ -224,6 +224,28 @@ def test_ax_logging(tmpdir: Path, cmd_arg: str, expected_str: str) -> None:
     assert "polynomial.z: fixed=10" in result
 
 
+@pytest.mark.parametrize(
+    "cmd_arg",
+    [
+        ("polynomial.coefficients=choice([-1, 0, 1],[2, 3, 4],[5, 6, 7])"),
+    ],
+)
+def test_jobs_using_choice_between_lists(tmpdir: Path, cmd_arg: str) -> None:
+    cmd = [
+        sys.executable,
+        "tests/apps/polynomial_with_list_coefficients.py",
+        "-m",
+        "hydra.run.dir=" + str(tmpdir),
+        "hydra.sweeper.ax_config.max_trials=3",
+    ] + [cmd_arg]
+    result = subprocess.check_output(cmd).decode("utf-8").rstrip()
+    assert (
+        "polynomial.coefficients: choice=['[-1,0,1]', '[2,3,4]', '[5,6,7]']" in result
```

Comment:
How am supposed to know what this is testing?
what does the fact that the choice appears in the output means about what is actually happening?
maybe it ran one job with ta single list?